lib/services/resource.go

```diff
@@ -213,7 +213,7 @@ type Ref struct {
 
 // IsEmpty checks whether the provided resource name is empty
 func (r *Ref) IsEmpty() bool {
-	return r.Name == ""
+	return r.SubKind == "" && r.Name == ""
```

Comment:
this no longer matches the godoc and could lead to unintended consequences
how about just not using this helper at all and checking desired fields directly as needed?

Reply:
Done.